libs/api/domains/cms/src/lib/cms.resolver.ts

```diff
@@ -254,17 +250,6 @@ export class CmsResolver {
     return this.cmsContentfulService.getOrganizationTags(input?.lang ?? 'is-IS')
   }
 
-  // DEPRECATED: will be removed after 18-05-21
-  @Directive(cacheControlDirective())
-  @Query(() => FrontpageSliderList, { nullable: true })
-  getFrontpageSliderList(
-    @Args('input') input: GetFrontpageSliderListInput,
-  ): Promise<FrontpageSliderList | null> {
-    return this.cmsContentfulService.getFrontpageSliderList(
-      input?.lang ?? 'is-IS',
-    )
-  }
-
```

Comment:
Seems like this might be in use in the `service-portal` here:
`libs/service-portal/graphql/src/lib/queries/getFrontpageSlides.ts`
We should probably sync with the and move them to the new way of fetching this data

Reply:
Ah very good point, I missed this.
Seems like that query is also outdated in regards to the "animationJsonAsset".
Yes, we should sync with the service-portal-team.

Reply:
The queries should be mostly compatible with the new frontpage query:
https://github.com/island-is/island.is/blob/main/apps/web/screens/queries/Frontpage.ts#L35-L52

Reply:
Thank you :+1: